tests/restapi/test_restapi.py

```diff
@@ -106,6 +106,8 @@ def test_data_path(construct_url, vlan_members):
     pytest_assert(r.status_code == 204)
 
     # Verify routes
+    # Add some delay before query
+    time.sleep(5)
```

Comment:
Why do we need this? We are not seeing any failure on Mellanox platform at the moment.

Reply:
The case is consistently failing on SN4600.

Reply:
@sumukhatv , the get request here is taking 1.2 second. Seems like we need to investigate this:

```
Jul 15 02:03:10.442687 str-msn4600c-acs-02 INFO restapi#supervisord: restapi [ trace ] 2021/07/15 02:03:10 release server write lock
Jul 15 02:03:10.442687 str-msn4600c-acs-02 INFO restapi#supervisord: restapi [  info ] 2021/07/15 02:03:10 request: duration 1.237732764s
```
A previous SET request is just 2ms:
```
Jul 15 02:03:09.188731 str-msn4600c-acs-02 INFO restapi#supervisord: restapi [  info ] 2021/07/15 02:03:09 request: duration 2.778701ms
```